mypy/plugin.py

```diff
@@ -170,12 +170,12 @@ def fail(self, msg: str, ctx: Context, *, code: ErrorCode | None = None) -> None
         raise NotImplementedError
 
     @abstractmethod
-    def named_type(self, name: str, args: list[Type]) -> Instance:
+    def named_type(self, fullname: str, args: list[Type], /) -> Instance:
```

Comment:
Changing the argument to `fullname` does make sense. Just not sure about adding `/` though. It feels a bit like an escape hatch. For example it hides the fact that the argument is still named something different in the implementation. I'd suggest removing the `/` here.

https://github.com/python/mypy/blob/ac6151ae6657a8bbd9793c74691c9985dae02c15/mypy/typeanal.py#L1939-L1941